package.json

```diff
@@ -20,15 +17,16 @@
     "test:conditional": "node ./node_modules/mendix-widget-build-script/dist/CheckPullRequestFromFork.js && echo 'Skip tests on forked pull request' || npm run deployAndTest",
     "test:dev": "karma start",
     "test:unit": "karma start --single-run --codeCoverage && node tests/remap.js",
-    "test:e2e": "tsc -p ./tests/e2e/tsconfig.json && wdio ./tests/e2e/wdio.conf.js",
-    "test:e2e:dev": "tsc -p ./tests/e2e/tsconfig.json && cross-env DEBUG=true wdio ./tests/e2e/wdio.conf.js",
+    "test:e2e": "pluggable-widgets-tools test:e2e:web",
+    "test:e2e:dev": "pluggable-widgets-tools test:e2e:web:dev",
     "lint": "tslint -c tslint.json '**/*.ts' '**/*.tsx' --exclude '**/node_modules/**'",
     "lint:fix": "npm run lint -- --fix",
     "deploy": " npm run updateProject && npm run deployApp",
     "build": "cross-env NODE_ENV=production grunt release",
     "updateProject": "node ./node_modules/mendix-widget-build-script/dist/UpdateMxProject.js",
     "deployApp": "node ./node_modules/mendix-widget-build-script/dist/DeployMxApp.js",
-    "deployAndTest": "npm run deploy && npm run test:e2e"
+    "deployAndTest": "npm run deploy && npm run test:e2e",
+    "postinstall": "patch-package"
```

Comment:
We're using old version of the node here, so I need to apply a patch here because there's an error in the package version function on the e2e js script.